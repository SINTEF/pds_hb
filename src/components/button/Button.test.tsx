import React from 'react'
import renderer from 'react-test-renderer'
import { Button } from '.'

it('renders without crashing', () => {
  const tree = renderer
    .create(
      <Button
        onClick={() => {
          return
        }}
      />
    )
    .toJSON()
  expect(tree).toMatchSnapshot()
})
