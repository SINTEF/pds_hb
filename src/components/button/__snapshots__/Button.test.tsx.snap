// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="react-ripples"
  onClick={[Function]}
  style={
    Object {
      "display": "inline-flex",
      "overflow": "hidden",
      "position": "relative",
    }
  }
>
  <button
    className="button medium standard"
    style={
      Object {
        "backgroundColor": undefined,
      }
    }
    type="button"
  >
    <i
      className="material-icons icon"
    >
      chevron_right
    </i>
  </button>
  <s
    style={
      Object {
        "borderRadius": "50%",
        "height": 35,
        "opacity": 0,
        "pointerEvents": "none",
        "position": "absolute",
        "transform": "translate(-50%, -50%)",
        "width": 35,
      }
    }
  />
</div>
`;
